src/commcare_cloud/terraform/modules/servers/main.tf

```diff
@@ -8,6 +8,9 @@ resource aws_instance "server" {
   key_name                = "${var.key_name}"
   vpc_security_group_ids  = ["${var.security_group_options[lookup(var.servers[count.index], "network_tier")]}"]
   source_dest_check       = false
+  credit_specification {
```

Comment:
What does this mean?

Reply:
Yeah `cpu_credits` can be either "unlimited" or "standard"; when it's "standard" your instance type comes with a certain number of CPU bust credits that'll it'll use when necessary until they run out; with "unlimited", it'll use them as necessary until they run out, and then start charging $.05/hr for extra usage of burst CPU. See https://aws.amazon.com/ec2/instance-types/t3/ (and t2 has the same concept) for more info.